backend/grant/utils/enums.py

```diff
@@ -18,6 +24,16 @@ class ProposalStatusEnum(CustomEnum):
 ProposalStatus = ProposalStatusEnum()
 
 
+class ProposalSortEnum(CustomEnum):
+    NEWEST = 'NEWEST'
+    OLDEST = 'OLDEST'
+    MOST_FUNDED = 'MOST_FUNDED'
+    LEAST_FUNDED = 'LEAST_FUNDED'
```

Comment:
I'm honestly OK with killing these, I don't know that there's a compelling use case. I would be down for one more sort though, which is something like ENDING_SOONEST. Might be nice to find proposals that are about to end and need to be pushed into funding.